### PR TITLE
Fixes ped bugs

### LINKED ISSUE #149: [Bug] Wrong invalid command result displayed
After keying in a invalid 'onduty' command, a invalid 'offduty' result is displayed instead.

<hr><sub>[original: Cronyxx/ped#4]<br/>
</sub>

### LINKED ISSUE #152: [Bug] Wrong command result given.
Typed in 'break' but 'next' command suggested instead.

<hr><sub>[original: Cronyxx/ped#7]<br/>
</sub>

### LINKED ISSUE #153: Wrong command result for invalid syntax
When typing "break" and enter, the command result is 

Invalid command format! 
next: Allocates next patient in queue to a room. Parameters: INDEX (must be a positive integer) 
Example: next 1

which belongs to the next command. So user does not know what they typed wrongly.

<hr><sub>[original: sreesubbash/ped#1]<br/>
</sub>

### LINKED ISSUE #156: Able to enqueue patient again who is being served by doctor
this person is in queue and being served :O

<hr><sub>[original: sreesubbash/ped#4]<br/>
</sub>

### LINKED ISSUE #158: Incorrect representation
I just typed resume 1. Doctor did not go on break previously. Hardcode and wrong response message

<hr><sub>[original: sreesubbash/ped#6]<br/>
</sub>

### LINKED ISSUE #163: Incorrect parsing of command offduty
There is nobody to `offduty` but the syntax is right

<hr><sub>[original: sreesubbash/ped#11]<br/>
</sub>

### LINKED ISSUE #179: Enqueue & next feature interaction
The same patient that is being served by a doctor can be added to the queue. Multiple instances of the same patient will exist.

Perhaps consider removing the patient from the patient list when enqueuing them/make them unable to be enqueued while assigned to a doctor .

<hr><sub>[original: alexwxh/ped#1]<br/>
</sub>

### PR DESCRIPTION
**Pull request by wongsm7**
Fixes #149 
Fixes #152 
Fixes #153 
Fixes #156   
Fixes #158 
Fixes #163 
Fixes #179

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/queue/AddConsultationRoomCommand.java [hunk 1]
```diff
@@ -19,10 +19,10 @@
     public static final String COMMAND_WORD = "onduty";
     public static final String MESSAGE_INVALID_DOCTOR_ID = "The reference ID '%1$s' does not belong "
             + "to any registered doctor!";
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a new room. "
-            + "Parameters: "
-            + "REFERENCE_ID \n"
-            + "Example: " + COMMAND_WORD + " 001A";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Marks a doctor as on-duty "
+            + "and adds it to the list of displayed rooms.\n"
```
Comment (SakuraBlossom) [inline src/main/java/seedu/address/logic/commands/queue/AddConsultationRoomCommand.java]: Please rephrase this. There should be no reference to any sort of room. It only adds a list of staff doctors on duty.

File: src/main/java/seedu/address/logic/commands/queue/BreakCommand.java [hunk 2]
```diff
@@ -17,7 +17,7 @@
     public static final String COMMAND_WORD = "break";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Doctor goes for a break\n"
+            + ": Doctor goes for a break. Avoids directing patients to the doctor based on the room number.\n"
```
Comment (SakuraBlossom) [inline src/main/java/seedu/address/logic/commands/queue/BreakCommand.java]: Please rephrase this. There should be no reference to any sort of room. It only adds a list of staff doctors on duty.

File: src/main/java/seedu/address/logic/commands/queue/ResumeCommand.java [hunk 3]
```diff
@@ -17,13 +17,14 @@
...
-            + ": Doctor is back from his/her break\n"
+            + ": Doctor is back from his/her break. Allows patients to be directed to the doctor "
+            + "based on the room number.\n"
             + "Parameters: INDEX (must be a positive integer)\n"
             + "Example: " + COMMAND_WORD + " 1";
 
     public static final String MESSAGE_SUCCESS = "Doctor %s resumes his/her duty";
     public static final String MESSAGE_DUPLICATE_ROOM = "This room already exists in the list.";
-    public static final String MESSAGE_ALREADY_BACK_FROM_BREAK = "Doctor is already back from his/her break";
+    public static final String MESSAGE_NOT_ON_BREAK = "Doctor isn't on break";
```
Comment (SakuraBlossom) [inline src/main/java/seedu/address/logic/commands/queue/ResumeCommand.java]: Use "is not" instead

File: src/main/java/seedu/address/model/queue/QueueManager.java [hunk 4]
```diff
@@ -77,10 +77,26 @@ public void poll() {
...
     }
 
+    /**
+     * Checks if a patient is being served
+     *
+     * @param id of the patient being served
+     * @return boolean which indicates whether the patient is being served
+     */
+    public boolean hasIdInRooms(ReferenceId id) {
+        UniqueElementList<ReferenceId> patientsBeingServed = new UniqueElementList<>();
```
Comment (SakuraBlossom) [inline src/main/java/seedu/address/model/queue/QueueManager.java]: Please rewrite this function. Why are you constructing another uniqueElementList?

File: src/main/java/seedu/address/model/queue/Room.java [hunk 5]
```diff
@@ -43,6 +43,10 @@ public ReferenceId getDoctor() {
         return patientCurrentlyBeingServed;
     }
 
+    public boolean hasPatientBeingServed() {
+        return !patientCurrentlyBeingServed.equals(Optional.empty());
```
Comment (SakuraBlossom) [inline src/main/java/seedu/address/model/queue/Room.java]: Use the isPresent() function instead

Comment (wongsm7) [reply src/main/java/seedu/address/model/queue/Room.java]: Ok changed

File: src/main/java/seedu/address/logic/commands/queue/AddConsultationRoomCommand.java (hunk 1 above)
Comment (wongsm7) [reply src/main/java/seedu/address/logic/commands/queue/AddConsultationRoomCommand.java]: Rephrased

File: src/main/java/seedu/address/logic/commands/queue/BreakCommand.java (hunk 2 above)
Comment (wongsm7) [reply src/main/java/seedu/address/logic/commands/queue/BreakCommand.java]: Rephrased

File: src/main/java/seedu/address/logic/commands/queue/ResumeCommand.java (hunk 3 above)
Comment (wongsm7) [reply src/main/java/seedu/address/logic/commands/queue/ResumeCommand.java]: Edited.

File: src/main/java/seedu/address/model/queue/QueueManager.java (hunk 4 above)
Comment (wongsm7) [reply src/main/java/seedu/address/model/queue/QueueManager.java]: Ok changed.

File: src/main/java/seedu/address/logic/commands/queue/ResumeCommand.java [hunk 6]
```diff
@@ -17,13 +17,14 @@
     public static final String COMMAND_WORD = "resume";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Doctor is back from his/her break\n"
+            + ": Doctor is back from his/her break. Allows patients to be directed to the doctor "
+            + "based on the room number.\n"
```
Comment (SakuraBlossom) [inline src/main/java/seedu/address/logic/commands/queue/ResumeCommand.java]: Please remove this line. No reference to room numbers please.

Comment (wongsm7) [reply src/main/java/seedu/address/logic/commands/queue/ResumeCommand.java]: Removed.

File: src/main/java/seedu/address/logic/commands/NextCommand.java [hunk 7]
```diff
@@ -20,7 +20,7 @@
     public static final String MESSAGE_DOCTOR_ON_BREAK = "Doctor is currently on break";
     public static final String MESSAGE_PERSON_NOT_IN_QUEUE = "This person '%1$s' is not in the queue";
     public static final String MESSAGE_DUPLICATE_ROOM = "This room already exists in the list.";
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Allocates next patient in queue to a room. "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Allocates next patient in queue to a room.\n"
```
Comment (SakuraBlossom) [inline src/main/java/seedu/address/logic/commands/NextCommand.java]: Please remove reference to a room

File: src/main/java/seedu/address/logic/commands/queue/BreakCommand.java [hunk 8]
```diff
@@ -17,7 +17,7 @@
     public static final String COMMAND_WORD = "break";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Doctor goes for a break\n"
+            + ": Doctor goes for a break. Marks the doctor as on break.\n"
```
Comment (SakuraBlossom) [inline src/main/java/seedu/address/logic/commands/queue/BreakCommand.java]: Rephrase to 'Marks the doctor as on break. Patients cannot be refered to the doctors on break.'

**Pull request merged**